Lesson07_Task01.py

```diff
@@ -0,0 +1,40 @@
+"""
+Home-task 07 Integers to english words
+
+# Convert a non-negative integer num to its English words representation.
+# Example 1:
+# Input: num = 123
+# Output: "One Hundred Twenty Three"
+
+# let's take that number always > 100 and only three digits
+# 100 <= num <= 999
+"""
+
+
+def numbertowords(num):
+    ones = ['', 'One', 'Two', 'Three', 'Four', 'Five',
+            'Six', 'Seven', 'Eight', 'Nine']
+    tens = ['', '', 'Twenty', 'Thirty', 'Forty', 'Fifty',
+            'Sixty', 'Seventy', 'Eighty', 'Ninety']
+    teens = ['Ten', 'Eleven', 'Twelve', 'Thirteen', 'Fourteen',
+             'Fifteen', 'Sixteen', 'Seventeen', 'Eighteen', 'Nineteen']
+
+    num_str = str(num)
+    length = len(num_str)
+
+    if length == 3:
```

Comment:
Here better to use not  two if   but if -> elif. Plus always try to minimize number of returns from function. Better to clloect data in to some result variable and only at the end use once` return result `